paimon-common/src/main/java/org/apache/paimon/CoreOptions.java

```diff
@@ -915,6 +915,13 @@ public class CoreOptions implements Serializable {
                     .defaultValue(MemorySize.parse("256 mb"))
                     .withDescription("Max memory size for lookup cache.");
 
+    public static final ConfigOption<Double> LOOKUP_CACHE_HIGH_PRIO_POOL_RATIO =
+            key("lookup.cache.high-prio-pool-ratio")
```

Comment:
I think lookup.cache.high-priority-pool-ratio may be better?

Reply:
fixed